ql/termstructures/volatility/optionlet/constantoptionletvol.hpp

```diff
@@ -35,29 +36,33 @@ namespace QuantLib {
     class ConstantOptionletVolatility : public OptionletVolatilityStructure {
       public:
         //! floating reference date, floating market data
-        ConstantOptionletVolatility(Natural settlementDays,
-                                    const Calendar& cal,
+        ConstantOptionletVolatility(Natural settlementDays, const Calendar &cal,
                                     BusinessDayConvention bdc,
-                                    const Handle<Quote>& volatility,
-                                    const DayCounter& dc);
+                                    const Handle< Quote > &volatility,
```

Comment:
Is your editor passing the code through some kind of automatic formatter? It's not a big problem, but it makes real changes a bit harder to spot.